suite_report.py

```diff
@@ -2061,8 +1976,7 @@ def print_report(self):
                 suite_dir = "--cylc_suite_dir--"
             trac_log.extend(
                 [
-                    "There has been an exception in "
-                    + "SuiteReport.print_report()",
+                    "There has been an exception in " + "SuiteReport.print_report()",
```

Comment:
Unification on this string.

```suggestion
                    "There has been an exception in SuiteReport.print_report()",
```